recipes/dbus/1.x.x/conanfile.py

```diff
@@ -67,9 +67,9 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("expat/2.5.0")
+        self.requires("expat/2.6.0")
```

Comment:
It will break Qt5 and Qt6. Both are using expat 2.5.0 and require dbus.

Reply:
@uilianries  Wayland also currently breaks them... just not be default with Wayland disabled. Would you rather I downgrade expat in Wayland to 2.5.0? Or open PRs to bump dependencies in Qt 5 and 6?

Reply:
xkbcommon requires expat 2.6.0, so qt is already using expat/2.6.0 (except if qtwebengine is activated, then there is already a conflict with expat : https://github.com/conan-io/conan-center-index/blob/41876ae1b94e61e249088e43ffcf551be3c9643f/recipes/qt/6.x.x/conanfile.py#L370-L371)

Reply:
I went ahead and opened #22929 to bump expat to 2.6.0 in Qt.

Reply:
#22929 has been merged, so this should be good to go now.